todooperation/src/Todo.js

```diff
@@ -0,0 +1,142 @@
+import React from "react";
+import { AddTodo } from "./AddTodo";
+import { ListTodo } from "./ListTodo";
+import axios from "axios";
+
+export class Todo extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            desc: "",
+            todos: []
+        };
+        this.id = 0;
+        this.isUpdate = false;
+        this.tempId = 0;
+    }
+
+    componentDidMount() {
+        axios({
+            method: "get",
+            url: "http://localhost:5000/todos"
```

Comment:
hardcoded URL are okay for now. but we can create constants for base URL and derive specific endpoints.